packages/core/src/VNodes/VNode.ts

```diff
@@ -47,11 +47,17 @@ export class VNode {
     get name(): string {
         return this.constructor.name;
     }
+    /**
```

Comment:
@dmo-odoo In case you wonder what the concrete use is of this, and what it has to do with the PR: it allows us to remove the reference to `LineBreakNode` in `DevTools`. It might be overkill though so if you want to skip it I'm not going to fight ;-)